--- /etc/dhcp/dhclient.conf	2017-08-08 13:09:23.835919413 +0100
+++ /etc/dhcp/dhclient.conf	2017-08-08 13:09:55.136310923 +0100
@@ -19,6 +19,9 @@
 	netbios-name-servers, netbios-scope, interface-mtu,
 	rfc3442-classless-static-routes, ntp-servers;

+# If you want to block the DNS Servers to use only google
+supersede   domain-name-servers 8.8.8.8,8.8.4.4;
+
 #send dhcp-client-identifier 1:0:a0:24:ab:fb:9c;
 #send dhcp-lease-time 3600;
 #supersede domain-name "fugue.com home.vix.com";
